src/test/java/org/fisco/bcos/channel/test/protocol/EnumNodeVersionTest.java

```diff
@@ -0,0 +1,135 @@
+package org.fisco.bcos.channel.test.protocol;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.fisco.bcos.channel.protocol.ChannelPrococolExceiption;
+import org.fisco.bcos.fisco.EnumNodeVersion;
+import org.junit.Test;
+
+public class EnumNodeVersionTest {
+
+    @Test
+    public void EnumNodeVersionTestFunc0() {
+        try {
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_0_0_RC1.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_0_0_RC2.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_0_0_RC3.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_0_0.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_0_1.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_1_0.getVersion()),
+                    is(true));
+        } catch (ChannelPrococolExceiption e) {
+            assertThat(true, is(false));
+        }
+    }
+
+    @Test
+    public void EnumNodeVersionTestFunc1() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'EnumNodeVersionTestFunc1' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4849551)